### DISCUSSION TITLE
new version 10 without spacing

### DISCUSSION THREAD

**Discussion by jonnypaulino** [primefaces/community#913] [PrimeReact] [answered]
When updating to version 10 in react, I noticed that the components had no spacing, did any configuration change?

it stayed that way

<img width="948" alt="image" src="https://github.com/primefaces/community/assets/62815240/b84dd70e-3210-4893-b1ee-441bc7d24c21">

before version 9.2.1 it was like this

<img width="946" alt="image" src="https://github.com/primefaces/community/assets/62815240/ec9d5d5b-f602-4c69-9d5e-287bacf7a0a4">

**Comment by melloware** [comment]
v10 Introduced CSS Layers so if you have custom CSS wrap it in `@layer primereact {` or if you are using a PR theme make sure to update your theme to match version 10.

**Reply by jonnypaulino** [reply]
How do I check if the theme is up to date? because I'm using the ready-made themes and still there's no spacing

**Reply by melloware** [reply, marked as answer]
Working OK for me with 10.3.1 using built in themes: https://stackblitz.com/edit/6mpsgr?file=src%2FApp.jsx

**Reply by mertsincan** [reply]
For now, closed. If the problem perstist, could you please create a Github issue with a sample stackblitz link to examine in detail? Thanks a lot for your understanding!

Best Regards,

**Reply by jonnypaulino** [reply]
hey guys. I managed to solve the problem using CSS Layers, thank you very much!
